docs/tutorials/hello_many_worlds.ipynb

```diff
@@ -915,6 +917,73 @@
         "From this plot you can see that the neural network has learned to overcome the systematic miscalibration."
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "C2RfWismj66S",
+        "colab_type": "text"
+      },
+      "source": [
+        "### 2.6 Verify outputs\n",
+        "Now use the trained model, to correct the qubit calibration errors. With Cirq:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "RoIlb7r7j5SY",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "def check_error(command_values, desired_values):\n",
+        "  \"\"\"Based on the value in `command_value` see how well you could prepare\n",
+        "  the full circuit to have `desired_value` when taking expectation w.r.t. Z.\"\"\"\n",
+        "  params_to_prepare_output = controller(command_values).numpy()\n",
+        "  full_circuit = noisy_preparation + model_circuit\n",
+        "\n",
+        "  # Test how well you can prepare a state to get expectation the expectation\n",
+        "  # value in `desired_values`\n",
+        "  for index in [0, 1]:\n",
+        "    state = cirq_simulator.simulate(\n",
+        "        full_circuit,\n",
+        "        {s:v for (s,v) in zip(control_params, params_to_prepare_output[index])}\n",
+        "    ).final_state\n",
+        "    expectation = z0.expectation_from_wavefunction(state, {qubit: 0}).real\n",
+        "    print(f'For a desired output (expectation) of {desired_values[index]} with'\n",
+        "          f' noisy preparation, the controller\\nnetwork found the following '\n",
+        "          f'values for theta: {params_to_prepare_output[index]}\\nWhich gives an'\n",
+        "          f' actual expectation of: {expectation}\\n')\n",
+        "\n",
+        "\n",
+        "check_error(commands, expected_outputs)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wvW_ZDwmsws6",
+        "colab_type": "text"
+      },
+      "source": [
+        "The value of the loss function during training will give you a rough idea of how close to expect the **expectation values of the full circuit controlled by `controller` to be to the desired outputs** in the above cell. If you aren't as concerned with the parameter values you can always check the outputs from above using `tfq`:"
```

Comment:
```suggestion
        "The value of the loss function—during training—provides a rough indication for how close to expect the expectation values of the full circuit, controlled by `controller`,  to the desired outputs in the above cell. If you aren't as concerned with the parameter values, you can always check the outputs using `tfq`:"
```
I tried to tighten up that first sentence but it's still pretty dense. Can you break it up into two?

Reply:
Broke it down. Thoughts ?